ocrd/ocrd/cli/workspace.py

```diff
@@ -244,10 +242,10 @@ def prune_files(ctx):
     with pushd_popd(workspace.directory):
         for f in workspace.mets.find_files():
             try:
-                if not exists(f.url):
+                if not f.local_filename or not exists(f.local_filename):
                     workspace.mets.remove_file(f.ID)
             except Exception as e:
-                log.debug(e)
+                log.warning("Error removing %f: %s", f, e)
```

Comment:
Maybe an error should be communicated as `ERROR`?

Reply:
I concur, so how about:
```python
log.exception("Cannot remove file '%s'", f.ID)
```

Reply:
Agreed, though I don't even remember when an exception can be raised here. But I'll let it throw and we'll see.